Tasks/Gradle/CodeAnalysis/Common/BaseTool.ts

```diff
@@ -78,6 +78,21 @@ export abstract class BaseTool implements IAnalysisTool {
         return tl.getBoolInput(this.uiInputName, false);
     }
 
+    protected findHtmlReport(xmlReport: string): string {
```

Comment:
Changed method access modifier from "private" to "protected" so that subclasses can override it, moved it upwards to reflect access modifier change.